python/README.md

```diff
@@ -1,4 +1,18 @@
-```py
+# Python automation
+This directory contains ready-to-use ArcGIS Notebooks (.ipynb).
```

Comment:
@kgonzago Do we have any docs we could reference here on Notebooks for those unfamiliar?

Reply:
Added [ArcGIS Notebooks](https://doc.arcgis.com/en/arcgis-online/get-started/components-of-the-notebook-editor.htm)